dereferencing/regs.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, pc=None, stack=None, frame=None,
         self.flagsr = flagsr
         self.args = args
         self.all = common + (frame, stack, pc, flagsr)
-        self.all = filter(None, self.all)
+        self.all = list(filter(None, self.all))
```

Comment:
on py3, filter returns a lazy generator, rather than a list with a length. so we explicitly eagerly realize the list here.